api/meter.go

```diff
@@ -163,18 +187,24 @@ func (r *relayMeter) loadData(from, to time.Time) error {
 
 	if updateDaily {
 		r.dailyUsage = dailyUsage
+
 		d := r.RelayMeterOptions.DailyMetricsTTL
 		if int(d.Seconds()) == 0 {
 			d = time.Duration(TTL_DAILY_METRICS_DEFAULT_SECONDS) * time.Second
 		}
+
 		r.dailyTTL = time.Now().Add(d)
 	}
+
 	if updateToday {
 		r.todaysUsage = todaysUsage
+		r.todaysLatency = todaysLatency
+
 		d := r.RelayMeterOptions.TodaysMetricsTTL
 		if int(d.Seconds()) == 0 {
 			d = time.Duration(TTL_TODAYS_METRICS_DEFAULT_SECONDS) * time.Second
 		}
+
```

Comment:
Same with this extra blank line, as well as previous ones in this function.

Reply:
Added these ones for readability; personally I find it much easier to read function when the logic is broken up a bit with some white space. Can remove if you disagree?

Reply:
I think this instance is fine to keep.